Added OVERRIDE_DEPENDENCIES flag

seanwiseman opened the pull request:
This will allow you to skip the installation of the required dependancies via an environment variable being present on the target OS and leave it down to the user to install them before hand.

In the long term I can extend the custom `cmdclass` that is already defined in the `setup.py` to allow similar behaviour. Though at present the `install_requires` list of dependancy strings is created outside of this class and applied directly to the `setup` function. This makes it difficult to integrate with so that logic could be moved into the `cmdclass` to tidy it up and allow for overriding.

seanwiseman committed:
Added OVERRIDE_DEPENDANCIES flag

giorgiosironi commented on setup.py:
```diff
@@ -55,6 +55,8 @@
 GROUP_DEFAULT = 'loris'
 GROUP_HELP = 'Group that will own the application and has permission to write to caches. [Default: %s]' % (USER_DEFAULT,)
 
+OVERRIDE_DEPENDENCIES = bool(os.environ.get('OVERRIDE_LORIS_DEPENDENCIES', False))
```
The title of the PR doesn't match the variable name, I think the variable name is the right one

seanwiseman replied:
Yes your right, have updated the PR name.

giorgiosironi commented:
Shouldn't this be opened against https://github.com/loris-imageserver/loris instead?

seanwiseman committed:
Allow dependencies to be read in from requirements.txt

This would allow easy overriding if you need to pin specific dependancy versions.
